### ISSUE TITLE
How to get the pydot graph of the following model?

### ISSUE THREAD

**Issue by yxchng** [BVLC/caffe#5375]
https://github.com/rbgirshick/py-faster-rcnn/blob/master/models/coco/VGG16/faster_rcnn_end2end/train.prototxt

I understand that there are a lot of discussion going on but i have tried all of them and none of them work. Is there any reason why there are so many people having this problem and yet it is still not updated.

Error message as follows:
TypeError: %d format: a number is required, not google.protobuf.pyext._message.RepeatedScalarContainer

This is the same as another github issue but the solutions suggested just don't work.

**Comment by yxchng** [comment]
Or sometimes it gives 

AttributeError: 'google.protobuf.pyext._message.RepeatedScalarConta' object has no attribute '_values'

**Comment by yxchng** [comment]
Error messages are different because i tried the different fixes suggested but all don't work

**Comment by zhaodaolimeng** [comment]
Same issue, but I managed to fixed it by rewrite line 94,96,98,107,109,111 with the following code:
``` python
layer.convolution_param.kernel_size[0],
layer.convolution_param.stride[0] if len(layer.convolution_param.stride) > 0 else 1,
layer.convolution_param.pad[0] if len(layer.convolution_param.pad) else 0)

layer.pooling_param.kernel_size,
layer.pooling_param.stride,
layer.pooling_param.pad)
```
Hope it helps.

**Comment by shelhamer** [comment]
Fixed by #5477 and #5487

**Comment by kritikaksu** [comment]
Thanks zhaodaolimeng. It works for me.
